fuel_test/clear_env.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -eu
+
+. localrc
+
+[ $DEBUG == "on" ] && set -x
+
+function clear_xs {
+	local xs_host="$1"
+	ssh -qo StrictHostKeyChecking=no root@$xs_host \
```

Comment:
This is good to clean up from remote machine. But for CI testing, the script will be run on the node(dom0) directly. No ssh is needed.